test/FastTests/Graph/GraphPermissionTests.cs

```diff
@@ -83,6 +83,7 @@ public void MultiplePathsWillAllow(string uid, string name)
             using (var store = GetDocumentStore())
             {
                 CreateData(store);
+                WaitForUserToContinueTheTest(store);
```

Comment:
Please create a test for this and remove this line

Reply:
*facepalm*
will do.